static/js/groups_page.js

```diff
@@ -89,8 +92,15 @@ var groupsPage = {
                 },
                 errorCallback = function (xhr, status, errorThrown) {
                     // This method is called when  error occurs while adding group.
-                    alert(xhr.responseText);
+                    if(xhr.responseJSON.error_message.includes('duplicate key')) {
+                        alert('Group name ' + group['name'] + ' already exists!')
+                    }
+                    else {
+                        alert(xhr.responseText);
+                    }
                 });
+
+            $('#CreateGroupBtn').attr('disabled', false);
```

Comment:
@HitRam What are the different scenarios when this line will be called?

Reply:
This line will be called only in case of errorCallBack.
Should I move this line inside the errorCallBack function?

Reply:
I think you should move it inside errorCallBack function!